rfcs/0165-Anonymous-scopes.md

```diff
@@ -0,0 +1,48 @@
+# RFC 165 - Anonymous scopes
+* Comments: [#165](https://github.com/taskcluster/taskcluster-rfcs/pull/165)
+* Proposed by: @ricky26
+
+# Summary
+
+We currently assign no scopes to calls with no credentials. With this proposal
+we would assign a single role to all calls and any scopes expanded from that.
+Additionally, we will move all API calls behind at least one scope.
+
+## Motivation
+
+This is primarily aimed at private TaskCluster installations. With this, it will
+be possible to prevent any useful access to the TaskCluster installation without
+credentials.
```

Comment:
You had mentioned the UI in our conversation -- is it OK that the UI is visible even without sign-in, just not showing any data?

Reply:
I think it's probably fine to have everything render just with no content / an error for this RFC. Perhaps this could be expanded later and we could have some finer-grained controls over hiding bits of the UI you don't have the respective view permission for but that seems like the icing on the cake to me.